src/pages/deriv-go/components/_tabs.tsx

```diff
@@ -1,8 +1,9 @@
 import React, { useEffect, ReactElement } from 'react'
 import styled, { css, keyframes } from 'styled-components'
-import { Flex, Show } from 'components/containers'
+import { Flex } from 'components/containers'
 import { Text } from 'components/elements'
 import device from 'themes/device'
+import { Desktop, Mobile } from 'components/containers/visibility'
```

Comment:
This can be imported directly from `components/containers`

Reply:
ok